gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/baseline.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "org.hypertrace.gateway.service.v1.baseline";
+
+package org.hypertrace.gateway.service.v1.baseline;
+
+import "org/hypertrace/gateway/service/v1/gateway_query.proto";
+
+message BaselineEntitiesRequest {
+  string entity_type = 1;
+  sfixed64 start_time_millis = 2;
+  sfixed64 end_time_millis = 3;
+
+  repeated string entity_ids = 4;
+  repeated org.hypertrace.gateway.service.v1.common.FunctionExpression baseline_aggregate_request = 7;
+  repeated BaselineTimeAggregation baseline_metric_series_request = 8;
+}
+
+message BaselineEntitiesResponse {
+  repeated BaselineEntity baselineEntity = 1;
```

Comment:
Shouldn't these be in Snake Case? Like `baseline_entity`?

Reply:
done